btcec/go.mod

```diff
@@ -3,13 +3,9 @@ module github.com/btcsuite/btcd/btcec/v2
 go 1.17
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220111032746-97732e52810c
+	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.0
```

Comment:
👍

Reply:
FWIW, we only have a spew dep due to these two lines in the tests: https://github.com/btcsuite/btcd/blob/master/btcec/pubkey_test.go#L244-L245

Reply:
I think it's okay to have that `spew` dependency. Anything that's not in this repo isn't really a problem as a dependency.